crud/crud.cpp

```diff
@@ -58,11 +58,31 @@ crud::start()
                     }
                     return true;
                 });
-
-
         });
 }
 
+void
+crud::do_raft_task_routing(const bzn::message& msg, bzn::message& response)
+{
+    switch(raft->get_state())
+    {
+        case bzn::raft_state::candidate:
+            response = do_candidate_tasks(msg);
```

Comment:
Missing "this":
ie.
this->do_candidate_tasks();

Reply:
I think you may have been working with an older version, or maybe I didn't push correctly. I removed the switch statements and replaced them with lookup tables.
This is no longer an issue...